<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo3_3,QTableView&amp;QStandardItemModel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>tableView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>plainTextEdit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_openFile"/>
   <addaction name="act_SaveFile"/>
   <addaction name="act_ModelData"/>
   <addaction name="separator"/>
   <addaction name="act_AddItem"/>
   <addaction name="act_InsterItem"/>
   <addaction name="act_DelItem"/>
   <addaction name="separator"/>
   <addaction name="act_Left"/>
   <addaction name="act_Center"/>
   <addaction name="act_Right"/>
   <addaction name="act_FontBold"/>
   <addaction name="separator"/>
   <addaction name="act_Exit"/>
  </widget>
  <action name="act_openFile">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/open.bmp</normaloff>:/icons/images/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="act_SaveFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/save.bmp</normaloff>:/icons/images/save.bmp</iconset>
   </property>
   <property name="text">
    <string>另存文件</string>
   </property>
  </action>
  <action name="act_ModelData">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/export1.bmp</normaloff>:/icons/images/export1.bmp</iconset>
   </property>
   <property name="text">
    <string>模型数据</string>
   </property>
  </action>
  <action name="act_AddItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/append.bmp</normaloff>:/icons/images/append.bmp</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </action>
  <action name="act_InsterItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/306.bmp</normaloff>:/icons/images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="act_DelItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/delete.bmp</normaloff>:/icons/images/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="act_Left">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/508.bmp</normaloff>:/icons/images/508.bmp</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
  </action>
  <action name="act_Center">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/510.bmp</normaloff>:/icons/images/510.bmp</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="act_Right">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/512.bmp</normaloff>:/icons/images/512.bmp</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
  </action>
  <action name="act_FontBold">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/500.bmp</normaloff>:/icons/images/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="act_Exit">
   <property name="icon">
    <iconset resource="../Recource/res.qrc">
     <normaloff>:/icons/images/132.bmp</normaloff>:/icons/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Recource/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
